src/ClimaLand.jl

```diff
@@ -11,10 +11,12 @@ include("shared_utilities/general_utils.jl")
 include("shared_utilities/TimeVaryingInputs.jl")
 using .TimeVaryingInputs
```

Comment:
not for this PR, but should we have `using .ModuleName` instead of `include("path-to-module.jl")` for all the files we include in this file?

Reply:
instead of having both the include and the using statements?

Reply:
yeah

Reply:
https://discourse.julialang.org/t/how-to-include-my-module-file-as-a-package/75682/12

ill try it out now. if it works it's just a couple of changes and might get done faster than if it is an issue